Users/Install.rst

```diff
@@ -183,6 +183,7 @@ virtualenv with:
 
 Using coala With Docker
 =======================
+Don't know what docker is? Have a look at `What is Docker? <http://docs.coala.io/en/latest/Users/Docker_Image.html>`_
```

Comment:
Hey @rohanrajpal 
The rest of the PR is good, but replacing `... what docker is?` with  `... what Docker is?` would ensure that the changes are consistent with the rest of the document.

Reply:
Thanks for reviewing and pointing that out! I've made the necessary changes